libs/client/ClientSession/GarnetClientSessionMigrationExtensions.cs

```diff
@@ -18,11 +18,11 @@ namespace Garnet.client
     /// </summary>
     public sealed unsafe partial class GarnetClientSession : IServerHook, IMessageConsumer
     {
-        static readonly Memory<byte> AUTH = Encoding.ASCII.GetBytes("AUTH");
-        static readonly Memory<byte> SETSLOTSRANGE = Encoding.ASCII.GetBytes("SETSLOTSRANGE");
-        static readonly Memory<byte> MIGRATE = Encoding.ASCII.GetBytes("MIGRATE");
-        static readonly Memory<byte> DELKEY = Encoding.ASCII.GetBytes("DELKEY");
-        static readonly Memory<byte> GETKVPAIRINSLOT = Encoding.ASCII.GetBytes("GETKVPAIRINSLOT");
+        static ReadOnlySpan<byte> AUTH => "AUTH"u8;
+        static ReadOnlySpan<byte> SETSLOTSRANGE => "SETSLOTSRANGE"u8;
+        static ReadOnlySpan<byte> MIGRATE => "MIGRATE"u8;
+        static ReadOnlySpan<byte> DELKEY => "DELKEY"u8;
+        static ReadOnlySpan<byte> GETKVPAIRINSLOT => "GETKVPAIRINSLOT"u8;
```

Comment:
Why not leave these as `static readonly`?

Reply:
oh duh, it's property

Reply:
The RVA pattern relies on the constant values to wrapped in properties, which can't be marked with `readonly` modifier because it's lowered to initializer expression.